src/styles/main.scss

```diff
@@ -44,8 +44,27 @@ body {
 
         &--color {
             display: flex;
-            max-width: 1140px;
+            flex-wrap: wrap;
             margin: 0 auto;
+            padding-left: 15px;
```

Comment:
Consider adding $gutter: 30px and reusing it like ($gutter / 2) like in bootstrap so we can reuse it.
I know, it’s like recreating bootstrap less/scss logic again but better that nothing at the moment.